SERVER_VERSION

```diff
@@ -1 +1 @@
-1.11.1
```

Comment:
I think we should delete this file - this link is tracked in the server source now